Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ruby:2.7.0-alpine
 
-RUN apk --no-cache add build-base git
+RUN apk --no-cache add build-base postgresql-dev git
```

Comment:
I’d like to make this configurable since this adds time to each action run and the goal of this action is to not need to bundle (also I never asked why you need to bundle? Not dismissing this is an issue but you may not need to). 

Could we instead add a line to the config or environment for adding this package? Env variable may be easier since you can just reference it in `entrypoint.sh` and install the needed package(s) there. Or you can use ruby to run the command (there are a few examples in this project of doing that). Happy to chat more in slack/discord about this but I definitely would like for this to be a configurable installation bc I imagine the same issue exists with MySQL and other native dependencies you may need installed in the container. 

Hopefully that makes sense, lmk if I can elaborate. I’ll also be looking into this more.